Products/ZenModel/ZenPackLoader.py

```diff
@@ -458,24 +458,29 @@ def load(self, pack, app):
                         raise Exception('Could not find unique name for trigger: "%s".' % trigger_conf['name'])
 
                     log.info('Creating trigger: %s' % test_name)
-                    tf.createTrigger(test_name, uuid=trigger_conf['uuid'], rule=trigger_conf['rule'])
 
+                    try:
+                        tf.createTrigger(test_name, uuid=trigger_conf['uuid'].lower(), rule=trigger_conf['rule'])
+                    except ServiceResponseError as e:
+                        log.warning("Could not create trigger '%s': %s" % (test_name, e))
```

Comment:
You are changing the behavior on error here. Do you not wan't the install to end?